index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Profile</title>
+  <link href="style1.css" rel="stylesheet" type="text/css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+</head>
+
+  <body>
+    <header>
+      <section>
+        <div class="upper">
+          <div class="black"></div>
+          <div class="orange"></div>
+          <div class="gray"></div>
```

Comment:
do not use such class names as black, orange, gray. What if you want to change the color for block? in this way you need to change code in css and html files. And if you choose more abstract name this can lead to changing only one line of code in css